forge.yaml

```diff
@@ -1,7 +1,7 @@
 variables:
   models:
-    advanced_model: &advanced_model anthropic/claude-3.7-sonnet
-    efficiency_model: &efficiency_model anthropic/claude-3.5-haiku
+    advanced_model: &advanced_model claude-3-7-sonnet-20250219
+    efficiency_model: &efficiency_model claude-3-5-haiku-20241022
```

Comment:
This change is relevant only if the user is using anthropic APIs directly. Open Router requires the model names to be what it was before.

Reply:
I see, Yes I am using direct anthropic APIs, but then how do I make it work without this change?

Reply:
Reverted to existing change